src/__fixtures__/monospace.ts

```diff
@@ -0,0 +1,30 @@
+const markdown = `\
+##моноспейс##
+вот ##это## моно**спейс** хе##хе##хе
+[привет ##мир## мне ##хорошо##](href "подсказка")
+`;
+
+const skeleton = `\
+##%%%1%%%##
+%%%2%%% ##%%%3%%%## %%%4%%%**%%%5%%%** %%%6%%%##%%%7%%%##%%%8%%%
+[%%%9%%% ##%%%%10%%%## %%%11%%% ##%%%12%%%##](href "%%%13%%%")
```

Comment:
you have mistake inside of the fixture, please check your pull request carefully.

```
##%%%%10%%%##
```